.github/workflows/ci.yml

```diff
@@ -227,7 +310,14 @@ jobs:
   tests-macos:
     runs-on: macos-latest
     name: Tests macOS
+    env:
+      BAAS_URL: ${{ secrets.REALM_QA_URL }}
+      BAAS_CLUSTER: ${{ needs.baas_matrix.outputs.dart-macos-clusterName }}
+      BAAS_API_KEY: ${{ secrets.ATLAS_QA_PUBLIC_API_KEY }}
+      BAAS_PRIVATE_API_KEY: ${{ secrets.ATLAS_QA_PRIVATE_API_KEY }}
+      BAAS_PROJECT_ID: ${{ secrets.ATLAS_QA_PROJECT_ID}}
```

Comment:
Since we're adding those to all test jobs, perhaps we can move all of these except `BAAS_CLUSTER` in the workflow env section.

Reply:
Done.